### ISSUE TITLE
Helm chart updates needed

### ISSUE THREAD

**Issue by bridgetkromhout** [virtual-kubelet/azure-aci#57]
Hi, folks!

I corrected issues in the Helm-related parts of the README: https://github.com/virtual-kubelet/azure-aci/pull/56

I also noticed that when I tried to perform the installation, I ran into problems with syntax in some of the Helm chart files:

```
$ helm install "$RELEASE_NAME" "$CHART_URL"
Error: template: virtual-kubelet/templates/secret.yaml:8:7: executing "virtual-kubelet/templates/secret.yaml" at <(not .Values.apiserverCert) and (not .Values.apiserverKey)>: can't give argument to non-function not .Values.apiserverCert
```

It appears this `can't give argument to non-function` message might indicate the same problem as here: https://github.com/helm/helm/issues/7711#issuecomment-593437917

Fixing that in https://github.com/virtual-kubelet/azure-aci/blob/master/helm/templates/secrets.yaml#L8 might help make the Helm installation work correctly.

**Comment by MikkelHegn** [comment]
@mingqishao is working on this currently.

**Comment by feiskyer** [comment]
The issue has been fixed by #61
